### PR TITLE
Feature/wrapper popular

### PR DESCRIPTION
**Pull request by mrSelieznov**
#228 Task

Add popular wrapper for home

// add after service, context, component bar

### CODE REVIEW

**Review by mrSelieznov — COMMENTED** [review]
Added wrapper popular, other due to prettier

File: src/components/Home.tsx [hunk 1]
```diff
@@ -1,13 +1,13 @@
-import { useEffect, useState } from 'react';
+import { useEffect, useState } from "react";
 
-import WrapperFavorites from './WrapperFavorites';
-import SmallInfoCard from './SmallInfoCard';
-import MovieBanner from './MovieBanner';
-import Container from './Container';
-import Slider from './Slider';
-import Title from './Title';
+import WrapperFavorites from "./WrapperFavorites";
```
Comment (oleksandrstarov) [inline src/components/Home.tsx:3]: Use single quotes

File: src/components/WrapperPopular.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+import { ThemeContext, ThemeContextType } from './ThemeContext';
+
+import SmallInfoCard from './SmallInfoCard';
+import Container from './Container';
+import Slider from './Slider';
+import Title from './Title';
+
+import api from '../service/api';
+
+type MovieId = {
```
Comment (oleksandrstarov) [inline src/components/WrapperPopular.tsx:11]: WrapperFavorites already has such type - no need to clone it

File: src/components/WrapperPopular.tsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  id: number;
+}
+
+const WrapperPopular = () => {
+  const { theme }: ThemeContextType = useContext(ThemeContext);
+  const [movies, setMovies] = useState<[]>([]);
```
Comment (oleksandrstarov) [inline src/components/WrapperPopular.tsx:17]: `<[]>` is not a valid type

**Review by oleksandrstarov — CHANGES_REQUESTED** [review]

File: src/components/WrapperPopular.tsx [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+import SmallInfoCard from './SmallInfoCard';
+import Container from './Container';
+import Slider from './Slider';
+import Title from './Title';
+
+import api from '../service/api';
+import { MovieId } from '../components/WrapperFavorites'
+
+type MoviesDataType = [];
```
Comment (oleksandrstarov) [inline src/components/WrapperPopular.tsx:12]: this is not a valid type still

File: src/components/WrapperPopular.tsx [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+const WrapperPopular = () => {
+  const { theme }: ThemeContextType = useContext(ThemeContext);
+  const [movies, setMovies] = useState<MoviesDataType>([]);
```
Comment (oleksandrstarov) [inline src/components/WrapperPopular.tsx:16]: Should be type <Array<MoviesId>>

**Review by oleksandrstarov — CHANGES_REQUESTED** [review]

**Review by oleksandrstarov — APPROVED** [review]

**Pull request merged**